coalib/results/Diff.py

```diff
@@ -70,6 +72,25 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unidiff(cls, unified_diff):
```

Comment:
though the module name is `unidiff`, I would call it `from_unified_diff`. But up to you^^